config/initializers/google_cloud_storage.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "google/cloud/storage"
+require "json"
+
+file = Rails.root.join(
+  "config/credentials/mo-image-archive-service-account.json"
+)
+# For some reason the autotester on github will still try to load this file
+# even though it should presumably be in test mode.
+if File.exist?(file) && !Rails.env.test?
```

Comment:
Would you be happier with a guard clause
```ruby
return if !File.exist?(file) || Rails.env.test?
```

Reply:
I wouldn't be any happier, but if everyone else would be, I can change it!  There's something odd about returning from a config file... :)